moveit_core/background_processing/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(MOVEIT_LIB_NAME moveit_background_processing)
 
 add_library(${MOVEIT_LIB_NAME} SHARED src/background_processing.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
```

Comment:
Can't we set this as a global property? (like https://github.com/ros-planning/navigation2/pull/1704/files#diff-3f9675091693ca07439ad55ceba95229R48)

Reply:
@henningkayser PR #285 set's this as a global property